server/sonar-server/src/test/java/org/sonar/server/user/ws/SetHomepageActionTest.java

```diff
@@ -64,7 +67,7 @@
   @Before
   public void setUp() {
     when(homepageTypes.getTypes()).thenReturn(asList(PROJECT, ORGANIZATION, MY_ISSUES, MY_PROJECTS));
```

Comment:
suggestion: have a utility method that must be explicitly set in each test: `onSonarCloud`, `onSonarQube`

Reply:
i love it (no more relevant here, but quite fond of the idea)